packages/knex/src/query/QueryBuilder.ts

```diff
@@ -491,6 +498,12 @@ export class QueryBuilder<T extends AnyEntity<T> = AnyEntity> {
 
 }
 
+type KnexStringRef = Knex.Ref<string, {
```

Comment:
```suggestion
type KnexStringRef = Ref<string, {
```

(and import `Ref` from `knex` if its not yet there)